src/__tests__/lib/exampleData.js

```diff
@@ -444,6 +444,9 @@ export const makeOutboxMessage = (data: $Shape<$Diff<Outbox, {| id: mixed |}>>):
   const outputTimestamp = timestamp ?? makeTime() / 1000;
   return deepFreeze({
     ...outboxMessageBase,
+    /* $FlowFixMe[exponential-spread] - Yeah, we should probably deal
```

Comment:
Can you make a ticket to track this, or do it in this PR?

I guess it's not a huge deal, since this is just a problem with typechecking performance, not runtime performance, but it does seem like something we should fix.